src/test/java/reposense/authorship/FileInfoExtractorTest.java

```diff
@@ -84,6 +85,14 @@ public void extractFileInfos_directoryWithValidWhitelistedName_success() {
         Assert.assertTrue(isFileExistence(Paths.get(".gradle/anything.txt"), files));
     }
 
+    @Test
+    public void extractFileInfos_branchWithValidWhitelistedFormatName_success() {
+        GitCheckout.checkout(config.getRepoRoot(), BRANCH_WITH_VALID_WHITELISTED_FORMAT_NAME_BRANCH);
```

Comment:
Just would like to better manage the branch naming. 
Can you change the branch name to this format -> `PR_NUM-CLASS_NAME-TEST_NAME`?
So in this case would be `535-FileInfoExtractorTest-branchWithValidWhitelistedFormatName`. You can put the `.txt` behind if its necessary.

Maybe `FileName` is more appropriate than `FormatName` here. Add a short description if you have to.